src/views/authentication/ScopesPane/index.tsx

```diff
@@ -157,10 +223,82 @@ export function ScopePane() {
 				</Stack>
 			</ScrollArea>
 
+			<Modal
+				size="md"
+				opened={isRegistring}
+				onClose={registerHandle.close}
+				trapFocus={false}
+				title={
+					<ModalTitle>Register user</ModalTitle>
+				}
+			>
+				<Text>
+					Please fill out the following required fields to register a new user to the scope <Text span c="bright">{registerScope}</Text>.
+				</Text>
+
+				<Form onSubmit={registerUser}>
+					<Table mt="md">
+						<Table.Thead>
+							<Table.Tr>
+								<Table.Th w="50%">Scope field</Table.Th>
+								<Table.Th w="50%">Value</Table.Th>
+							</Table.Tr>
+						</Table.Thead>
+						<Table.Tbody>
+							{registerFields.map((field, i) => {
+								const fieldName = field.subject.toLowerCase();
+								const ValueInput = HIDDEN_SCOPE_FIELDS.has(fieldName)
```

Comment:
thought: maybe `SENSITVE_SCOPE_FIELDS` as hidden could be interprated as non visible?